dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -38,7 +39,34 @@
            ]
 
 
-class LeapHybridSampler(dimod.Sampler):
+class ClosableClientBaseMixin(AbstractContextManager):
```

Comment:
Should this be marked as an internal class (`_ClosableClientBaseMixin`)? It's not documented externally.

Reply:
Sure; I think this mixin is very ephemeral anyway. I'm currently leaning towards an ABC for "closable resources" in dimod (to address #555).